criu/fdstore.c

```diff
@@ -53,17 +52,8 @@ int fdstore_init(void)
 		return -1;
 	}
 
-	if (!opts.unprivileged || has_cap_net_admin(opts.cap_eff)) {
-		rcv_opt_name = SO_RCVBUFFORCE;
-		snd_opt_name = SO_SNDBUFFORCE;
-	} else {
-		rcv_opt_name = SO_RCVBUF;
-		snd_opt_name = SO_SNDBUF;
-	}
-
-	if (setsockopt(sk, SOL_SOCKET, snd_opt_name, &buf[0], sizeof(buf[0])) < 0 ||
-	    setsockopt(sk, SOL_SOCKET, rcv_opt_name, &buf[1], sizeof(buf[1])) < 0) {
-		pr_perror("Unable to set SO_SNDBUFFORCE/SO_RCVBUFFORCE");
+	if (sk_setbufs(sk, buf)) {
+		pr_perror("Unable to set socket snd/rcv buffer size");
```

Comment:
sk_setbufs->restore_opt->do_restore_opt already prints error messages, so we can probably remove pr_error here.

Reply:
Done, thanks.

Reply:
pr_error is still there after sk_setbufs error, probably you lost this hunk.